Extensions/TileMap/JsExtension.js

```diff
@@ -510,10 +570,12 @@ module.exports = {
         instance,
         associatedObject,
         pixiContainer,
-        pixiResourcesLoader
+        pixiResourcesLoader,
+        pixiRenderer
       );
 
-      this._pixiObject = new Tilemap.CompositeRectTileLayer(0);
+      pixiRenderer.plugins.tilemap = new Tilemap.TileRenderer();
```

Comment:
Why is this line needed? This will create a new renderer at the creation of each instance on the scene (remember, you may not have a single tilemap, probably multiple per levels).
So this sounds inefficient at best, very dangerous at worst.

Reply:
This unfortunately can not work otherwise. The guys at pixi-tilemap told me its required

see https://github.com/pixijs/tilemap/issues/124